### PR TITLE
created content for pull request for TT - Fragrance Dataset

### PR DESCRIPTION
**Pull request by nicolasfoss**
Thank you for your submission! Please leave this text as-is when you submit, and then check the checklist.

Usability:

- [x] This dataset has not already been used in TidyTuesday.
- [x] The dataset is less than 20MB when saved as a tidy CSV.
- [x] I can imagine at least one data visualization related to this dataset.

Preparation:

- [x] `cleaning.R` has code to download and clean the dataset, resulting in descriptively-named data.frames.
- [x] I ran `saving.R` to create `csv`s and `md` data dictionaries for each of my data.frames.
- [x] I filled in descriptions for each variable in each of the `{dataset}.md` files.
- [x] I edited the `intro.md` file to introduce my dataset.
- [x] I included at least one image for my dataset as a `png` file.
- [x] I completed the information in `meta.yaml`, including descriptive alt text for each image.
- [x] I provided information in `meta.yaml` about how to credit me, and deleted any parts of that block that I do not want you to use.

### CODE REVIEW

**Comment by jonthegeek** [comment]
@nicolasfoss Thank you so much for the submission! I'll review it later today!

FYI, on GitHub you can use an empty pair of square brackets in a bulleted list (like this `- [ ]` for an interactive checkbox, or you can fill it in with an x like `- [x]` for a checked box. I should probably spell that out in the template! Then GitHub tracks the items as "to-do" for that particular issue or PR. Eg:

- [ ] Describe how checkboxes work.
- [ ] Write a ticket to clean up the PR instructions.

**Comment by nicolasfoss** [comment]
I wondered about that! 

I will do that for the Dr. Nick dataset. I will probably submit that next week sometime. 

Thanks for all the instructions. It made everything easy to use. I use GitHub desktop so that's the route I went but it's basically the same thing.

**Comment by nicolasfoss** [comment]
@jonthegeek I fixed the checkbook for good measure. 

Have a great day!

**Comment by jonthegeek** [comment]
> @jonthegeek I fixed the checkboxes for good measure.

Isn't it more satisfying that way? 😁 

Thanks for the submission(s)! I'm trying to "finish" dataset curation for the year within the next few days, and your Dr. Nick dataset will leave just 1 more (besides the one I'm working on)! 🎉

File: data/curated/parfumo_fragrance/cleaning.R [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+### Thanks to Olga G. Miufana!
+###_____________________________________________________________________________
+
+# packages
+library(httr)
+library(tidyverse)
+library(jsonlite)
+library(glue)
+library(janitor)
+library(here)
```
Comment (jonthegeek) [inline data/curated/parfumo_fragrance/cleaning.R:13]:
```suggestion
library(naniar)
```

File: data/curated/parfumo_fragrance/intro.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+This week, we're diving into the fascinating world of fragrances with a dataset sourced from [Parfumo](https://www.parfumo.com/), a vibrant community of perfume enthusiasts.  [Olga G.](https://www.kaggle.com/olgagmiufana1) webscraped these data from the various fragrance sections on the above-referenced website. Here is a description from the author:
```
Comment (jonthegeek) [inline data/curated/parfumo_fragrance/intro.md:1]:
```suggestion
This week we're diving into the fascinating world of fragrances with a dataset sourced from [Parfumo](https://www.parfumo.com/), a vibrant community of perfume enthusiasts.  [Olga G.](https://www.kaggle.com/olgagmiufana1) webscraped these data from the various fragrance sections on the Parfumo website. Here is a description from the author:
```

File: data/curated/parfumo_fragrance/intro.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+> The data provides a comprehensive look at how various perfumes are rated, which families of scents they belong to, and detailed breakdowns of the key olfactory components that define their overall profile
+
+We’ll explore how perfumes are rated, uncover the scent families they belong to, and delve into the minds of the perfumers behind them. From the year of release to the delicate composition of each scent, this dataset offers a rich olfactory experience for anyone curious about the magic behind their favorite perfumes.
+
+Join us as we decode the stories within these perfumes, from the top notes that hit you first to the lasting base notes that linger in the air. Whether you're a fragrance aficionado or just curious about the data behind the scents, this exploration will open your eyes (and nose) to the artistry of perfume crafting. Ready to sniff out some data?
+
+Thank you to [Nicolas Foss](www.linkedin.com/in/nicolas-foss) this week's dataset.
```
Comment (jonthegeek) [inline data/curated/parfumo_fragrance/intro.md:11]:
```suggestion
```
You'll automatically be credited with a link to your GitHub profile.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
Just a few tweaks. I'll apply the changes myself, then assign this to a week. Thanks again!

**Comment by jonthegeek** [comment]
Oh, not quite! Please don't delete sections of the meta.yaml file! I'm adding an article (which really means "anything that talks about the data", so https://www.parfumo.com/Perfumes works), and making sure nothing else is missing.

File: README.md [hunk 4]
```diff
@@ -81,6 +81,7 @@ If you are using TidyTuesday to teach data-related skills, [please let us know](
 | 47 | `2024-11-19` | [Bob's Burgers Episodes](data/2024/2024-11-19/readme.md) | [bobsburgersR R Package](https://github.com/poncest/bobsburgersR) | [Bob’s Burgers Episode Fingerprints by Season](https://stevenponce.netlify.app/projects/standalone_visualizations/sa_2024-11-11.html) | 
 | 48 | `2024-11-26` | [U.S. Customs and Border Protection (CBP) Encounter Data](data/2024/2024-11-26/readme.md) | [U.S. Customs and Border Protection](https://www.cbp.gov/) | [U.S. Border Patrol Encounters](https://gdatascience.github.io/us_border_patrol_encounters/us_border_patrol_encounters.html) | 
 | 49 | `2024-12-03` | [National Highways Traffic Flow](data/2024/2024-12-03/readme.md) | [WebTRIS Traffic Flow API](https://webtris.nationalhighways.co.uk/api/swagger/ui/index) | [National Highways](https://webtris.nationalhighways.co.uk/) | 
+| 50 | `2024-12-10` | [The Scent of Data - Exploring the Parfumo Fragrance Dataset](data/2024/2024-12-10/readme.md) | [Parfumo Fragrance Dataset ](https://www.kaggle.com/datasets/olgagmiufana1/parfumo-fragrance-dataset) | [Perfumes at parfumo](https://www.parfumo.com/Perfumes) |
```
Comment (jonthegeek) [inline README.md:84]:
```suggestion
| 50 | `2024-12-10` | [The Scent of Data - Exploring the Parfumo Fragrance Dataset](data/2024/2024-12-10/readme.md) | [Parfumo Fragrance Dataset](https://www.kaggle.com/datasets/olgagmiufana1/parfumo-fragrance-dataset) | [Perfumes at parfumo](https://www.parfumo.com/Perfumes) | 
```

File: data/2024/2024-12-10/meta.yaml [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
+title: "The Scent of Data - Exploring the Parfumo Fragrance Dataset"
 article:
-  title: Perfumes at parfumo
-  url: https://www.parfumo.com/Perfumes
+  title: "Perfumes at parfumo"
+  url: "https://www.parfumo.com/Perfumes"
 data_source:
-  title: Parfumo Fragrance Dataset 
-  url: https://www.kaggle.com/datasets/olgagmiufana1/parfumo-fragrance-dataset
+  title: "Parfumo Fragrance Dataset "
```
Comment (jonthegeek) [inline data/2024/2024-12-10/meta.yaml:6]:
```suggestion
  title: "Parfumo Fragrance Dataset"
```

File: data/2024/readme.md [hunk 6]
```diff
@@ -53,3 +53,4 @@ Archive of datasets and articles from the 2024 series of `#TidyTuesday` events.
 | 47 | `2024-11-19` | [Bob's Burgers Episodes](2024-11-19/readme.md) | [bobsburgersR R Package](https://github.com/poncest/bobsburgersR) | [Bob’s Burgers Episode Fingerprints by Season](https://stevenponce.netlify.app/projects/standalone_visualizations/sa_2024-11-11.html) |  
 | 48 | `2024-11-26` | [U.S. Customs and Border Protection (CBP) Encounter Data](2024-11-26/readme.md) | [U.S. Customs and Border Protection](https://www.cbp.gov/) | [U.S. Border Patrol Encounters](https://gdatascience.github.io/us_border_patrol_encounters/us_border_patrol_encounters.html) |  
 | 49 | `2024-12-03` | [National Highways Traffic Flow](2024-12-03/readme.md) | [WebTRIS Traffic Flow API](https://webtris.nationalhighways.co.uk/api/swagger/ui/index) | [National Highways](https://webtris.nationalhighways.co.uk/) |  
+| 50 | `2024-12-10` | [The Scent of Data - Exploring the Parfumo Fragrance Dataset](2024-12-10/readme.md) | [Parfumo Fragrance Dataset ](https://www.kaggle.com/datasets/olgagmiufana1/parfumo-fragrance-dataset) | [Perfumes at parfumo](https://www.parfumo.com/Perfumes) |
```
Comment (jonthegeek) [inline data/2024/readme.md:56]:
```suggestion
| 50 | `2024-12-10` | [The Scent of Data - Exploring the Parfumo Fragrance Dataset](2024-12-10/readme.md) | [Parfumo Fragrance Dataset](https://www.kaggle.com/datasets/olgagmiufana1/parfumo-fragrance-dataset) | [Perfumes at parfumo](https://www.parfumo.com/Perfumes) |  
```

**Review by jonthegeek — APPROVED** [review]
This will do it. Thanks!

**Pull request merged**